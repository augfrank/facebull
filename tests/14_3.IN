M0   C0   C3  2601
M1   C0   C4  2196
M2   C0   C5  2511
M3   C0   C7  2211
M4   C0   C9  2641
M5   C0   C10  2228
M6   C1   C2  2300
M7   C1   C3  2275
M8   C1   C8  2623
M9   C1   C9  2056
M10   C1   C10  2978
M11   C2   C3  2017
M12   C2   C4  2358
M13   C2   C5  2491
M14   C2   C6  2037
M15   C2   C7  2380
M16   C2   C13  2607
M17   C3   C6  2268
M18   C3   C7  2320
M19   C3   C12  2017
M20   C4   C1  2296
M21   C4   C7  2766
M22   C4   C8  2034
M23   C4   C11  2417
M24   C4   C13  2285
M25   C5   C0  2257
M26   C5   C2  2851
M27   C5   C3  2848
M28   C5   C4  2435
M29   C5   C6  2330
M30   C5   C7  2149
M31   C5   C8  2204
M32   C5   C9  2348
M33   C5   C10  2038
M34   C6   C2  2362
M35   C6   C3  2044
M36   C6   C4  2084
M37   C6   C11  2077
M38   C6   C13  2853
M39   C7   C5  2928
M40   C7   C8  2623
M41   C8   C5  2039
M42   C8   C7  2856
M43   C8   C10  2951
M44   C8   C11  2035
M45   C8   C12  2600
M46   C9   C0  2315
M47   C9   C3  2837
M48   C9   C7  2324
M49   C9   C10  2834
M50   C9   C13  2373
M51   C10   C4  2207
M52   C10   C8  2445
M53   C10   C13  2825
M54   C11   C1  2755
M55   C11   C3  2042
M56   C11   C4  2920
M57   C11   C6  2773
M58   C11   C8  2773
M59   C12   C0  2666
M60   C12   C2  2148
M61   C12   C4  2915
M62   C12   C5  2043
M63   C12   C7  2934
M64   C12   C11  2475
M65   C12   C13  2504
M66   C13   C3  2516
M67   C13   C6  2060
M68   C13   C10  2387
M69   C13   C12  2890
