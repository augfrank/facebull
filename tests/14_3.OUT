30803
5 9 15 19 20 25 37 40 41 50 51 55 60 67
